.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ Changelog category (leave one):
 - Not for changelog (changelog entry is not required)
 
 
-Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
+## Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
```

Comment:
Maybe it is better to use `###`? To me `##` looks too big, especially for this line.

- `##`

## Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

- `###`

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Reply:
Yep, looks good to me

Reply:
```suggestion
### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
```